crates/matrix-sdk-base/src/rooms/mod.rs

```diff
@@ -95,6 +100,10 @@ pub struct BaseRoomInfo {
     pub(crate) tombstone: Option<MinimalStateEvent<RoomTombstoneEventContent>>,
     /// The topic of this room.
     pub(crate) topic: Option<MinimalStateEvent<RoomTopicEventContent>>,
+    /// All Minimal state events that containing one or more running matrixRTC
+    /// memberships.
+    #[serde(skip_serializing_if = "HashMap::is_empty", default)]
+    pub(crate) rtc_member: HashMap<OwnedUserId, MinimalStateEvent<CallMemberEventContent>>,
```

Comment:
Just realized that this is a `HashMap`. We generally avoid it, it's only more performant than `BTreeMap` for large numbers of items but has non-deterministic iteration order.